--- sessions_controller.rb.orig 2022-08-30 21:28:06.724046832 +0200
+++ sessions_controller.rb      2022-08-30 21:28:54.868342389 +0200
@@ -21,7 +21,11 @@

   def destroy
     logout
-    redirect_to login_url, :notice => I18n.t('sessions.logged_out')
+    if FoodsoftConfig[:logout_redirect_to].present?
+      redirect_to FoodsoftConfig[:logout_redirect_to]
+    else
+      redirect_to login_url, :notice => I18n.t('sessions.logged_out')
+    end
   end

   # redirect to root, going to default foodcoop when none given
